### PR TITLE
add script for ogs grid

### PR DESCRIPTION
**Pull request by jvollhueter**
In this PR is a script added with which you can create a grid out of given (geometric) parameters

### CODE REVIEW

**Comment by jvollhueter** [comment]
Please don't merge, script does not work

**Comment by jvollhueter** [comment]
works

File: Benchmarks/ExampleSetups/OGSExampleSetup/MakeGrid.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+                       "/bin/"
+subprocess.call(ogs_utilities_string + "ExtractSurface -i "
+                                       "my_first_model.vtu -o "
+                                       "right_boundary.vtu -x -1 -y 0 -z 0 -a 0.")
+subprocess.call(ogs_utilities_string + "ExtractSurface -i "
+                                       "my_first_model.vtu -o "
+                                       "left_boundary.vtu -x 1 -y 0 -z 0 -a 0.")
+subprocess.call(ogs_utilities_string + "ExtractSurface -i "
+                                       "my_first_model.vtu -o "
+                                       "top_boundary.vtu -x 0 -y 0 -z -1 -a 30.")
```
Comment (jbathmann) [inline Benchmarks/ExampleSetups/OGSExampleSetup/MakeGrid.py:294]: This part only runs with windows systems. In the docs, a description for ubuntu files can be found. Can you maybe add a check for the operating system in order to decide which part to use? Example can be found in OGSLargeScale3D.py

**Review by jbathmann — CHANGES_REQUESTED** [review]
See comment

**Comment by jvollhueter** [comment]
Now the script works with Windows and Linux based os. MacOS-User have to adopt it manually.

**Review by jbathmann — APPROVED** [review]
Looks good!

**Comment by jbathmann** [comment]
Any reason to wait with the merge?

**Comment by jvollhueter** [comment]
> Any reason to wait with the merge?

I don't think so. Should I merge it?

**Pull request merged**